src/main/lens-proxy.ts

```diff
@@ -131,7 +131,11 @@ export class LensProxy {
           }
         }
       }
-      res.writeHead(500).end("Oops, something went wrong.")
+      try {
+        res.writeHead(500).end("Oops, something went wrong.")
+      } catch (e) {
+        // Can't send headers. Probably app is closing.
```

Comment:
Should we log an error?

Reply:
I have a feeling that logger is going south also at this point, but does not hurt if that does not cause any further issues

Reply:
Are we writing logs to `/Users/ixrock/Library/Logs/Electron/logs/lens.log`, right?

Reply:
yes. Tested logging the error and at least on Windows it did not cause any further issues.